gae/room/protorpc_messages.py

```diff
@@ -991,6 +991,66 @@ class SiteCaptain(messages.Message):
   type = messages.StringField(4)
 
 
+############
+# StaffPosition #
+############
+def StaffPositionModelToMessage(mdl):
+  s = StaffPosition(
+      id=mdl.key.integer_id(),
+      name=mdl.position_name,
+    )
+  if mdl.hourly_rate_after_date:
+    s.hourly_rates = [{'date': str(dr[0]), 'rate': float(dr[1])} for dr in (
+          dr.split() for dr in mdl.hourly_rate_after_date if dr)]
+  else:
+    s.hourly_rates = [{'date': "", 'rate': mdl.hourly_rate}]
+  if mdl.mileage_rate_after_date:
+    s.mileage_rates = [{'date': str(dr[0]), 'rate': float(dr[1])} for dr in (
+                dr.split() for dr in mdl.mileage_rate_after_date if dr)]
+  else:
+    s.mileage_rates = [{'date': "", 'rate': mdl.mileage_rate}]
+  return s
+
+
+def StaffPositionMessageToModel(msg, mdl):
```

Comment:
This function can be very simple.  We are defining the API so we can have it use only the new fields hourly_rates and mileage_rates.